whiterabbit/src/main/java/org/ohdsi/whiteRabbit/DbSettings.java

```diff
@@ -26,7 +26,8 @@
 public class DbSettings {
 	public static int	DATABASE	= 1;
 	public static int	CSVFILES	= 2;
-	
+	public static int	SASFILES	= 3;

```

Comment:
Would it make sense to make this an enum? Alternatively, you could distinguish on “database” vs “file” as source type, and for each source type be able to select a format (db: postgres; file: csv)

Reply:
Not implementing this atm, would affect a lot of parts of the code